lib/bike_brigade_web/live/program_live/form_component.html.heex

```diff
@@ -74,13 +73,50 @@
             What we deliver for this program
           </:subtitle>
           <:actions>
-            <.button size={:xsmall} patch={~p"/programs/#{@program}/items/new"}>
+            <.button size={:xsmall} phx-click={JS.push("add_item", target: @myself)}>
               New Item
             </.button>
           </:actions>
         </.header>
 
-        <.table id="items" rows={@program.items}>
+        <.table id="items" rows={inputs_for(f, :items) |> Enum.with_index()}>
+          <:col :let={{item_form, i}} label="Name">
+            <.input type="hidden" field={{item_form, :id}} />
+            <.input type="hidden" field={{item_form, :program_id}} />
+            <.input type="text" field={{item_form, :name}} />
+          </:col>
+
+          <:col :let={{item_form, i}} label="Description" show_at={:large}>
+            <.input type="text" field={{item_form, :description}} />
+          </:col>
+
+          <:col :let={{item_form, i}} label="Category">
+            <.input
+              type="select"
+              field={{item_form, :category}}
+              options={Ecto.Enum.values(Delivery.Item, :category)}
+            />
+          </:col>
+
+          <:col :let={{item_form, i}} label="Hidden?">
+            <.input type="checkbox" field={{item_form, :hidden}} />
+          </:col>
```

Comment:
I'm not super clear on why this can be hidden — do you think it might confuse users as well?

Reply:
I changed the label to "archived?" which I think is clearer